dev/explainer.clj

```diff
@@ -39,7 +39,8 @@
 
 ;; ** def
 
-(def P (parse/parse-fn 'Plus {:kernel/link @wl/kernel-link-atom}))
+(def wolfPlus (parse/parse-fn 'Plus {:kernel/link @wl/kernel-link-atom}))
+(wolfPlus 1 2 3) ; => 6
```

Comment:
### Conventions
It would be cool to have some nice convention for this 
I sometimes use `:` for those kinds of things as a 'fake' ns, so this would end up being `W:Plus`—some people don't like though :) 

### API/Thinking out Loud
IIRC `w` in `(wl/load-all-symbols 'w)` will end up being the namespace for the loaded vars, would be interesting to see if it's possible to pass in a list of symbols. We would then have `wl/load-symbols` instead w/ two arities. By default load *all* symbols, if and arg is passed load only specified symbols, just a thought.

Reply:
Cool (scary? :-)), I didn't know you can use : in symbols. I like the proposal, and it would surely be good to have a simple convention around this. Regarding people - no matter what you do, someone will mislike it 🤷 :)

I guess the motivation for load-symbols would be speed, when loading just a few? We might also instead support a regexp (if that would be performant with Wolfram). 
A thought I had was to make a ns at build-time where we'd intern symbols and docs, something like `wolframite.wolfram-lang-preloaded`, that ppl could use instead of loading all syms themselves - of course with the risk of missing newer symbols and updated docs.